recipes/wavespectra/meta.yaml

```diff
@@ -0,0 +1,88 @@
+{% set name = "wavespectra" %}
+{% set version = "3.7.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8ae6cf102ff3ef9bfb94bbc5842533746336489a97d9233cee5ba7f0f60681d1
+  
+
+build:
+  # Uncomment the following line if the package is pure Python and the recipe is exactly the same for all platforms.
+  # It is okay if the dependencies are not built for all platforms/versions, although selectors are still not allowed.
+  # See https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python for more details.
+  # noarch: python
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    # Compilers are named 'c', 'cxx' and 'fortran'.
+    - {{ compiler('fortran') }}
+    - numpy
+  host:
+    - python
+    - pip
+    - numpy
+    - click
+    - cmocean
+    - dask
+    - gcsfs
+    - fsspec
+    - matplotlib-base
+    - numpy
+    - pandas
+    - python-dateutil
+    - pyyaml
+    - sortedcontainers
+    - scipy
+    - toolz
+    - xarray
+    - zarr
+    - pytest-runner
```

Comment:
It's likely that many of these aren't needed in `host`. I would start from `numpy`, `pytest-runner`, `python` and `pip` and expand from there.

Reply:
@chrisburr , thanks for your review.
Didn't want to abuse your system with many trials-and-error runs. I guessed that when pytest-runner is used then some of the pytests will be run which would require the packages.

Reply:
IIRC `pytest-runner` is just used to make `python setup.py test` work. It definitely doesn't cause the unit tests to be executed.

As `pytest-runner` is included in `setup_requires` of the `setup.py` it is actually needed even if it doesn't do anything.

Reply:
ok, thanks for explaining. I removed most of the packages. Indeed only pytest-runner was required.